server/controllers/match.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose')
+
+// add a match and return match ID
+function addMatch() {
+
+  var Match = mongoose.model('matches');
```

Comment:
This function won't access the model, it will create a new blank model

Reply:
Changed it so that it creates a new match document, with the match creator as its first userID and responds with the match id for that game.